testing/workflows/components/unit_tests.jsonnet

```diff
@@ -106,6 +106,26 @@ local dagTemplates = [
     ]),  // create-pr-symlink
     dependencies: ["checkout"],
   },  // create-pr-symlink
+  {
+    template: buildTemplate("flake8-test", [
+      "python",
+      "-m",
+      "testing.test_flake8",
+      "--test_files_dirs=" +
+      srcDir + "/kubeflow/application/tests" + "," +
```

Comment:
I think we can just do srcDir + "/kubeflow"? It should recursive walk the tree right?

Reply:
After looking at:

https://github.com/kubeflow/kubeflow/pull/2247/commits/2337b2896a093521f8578fde65f2faf7cc515727#diff-e75784592f0b3b405baa4710e7a10ce6R25

I thought there might be a reason to specify sub-directories explicitly.

Yes, it should be a recursive walk. I have implemented the changes.

Also, we might need to propagate these changes here:
https://github.com/kubeflow/kubeflow/blob/master/testing/workflows/components/unit_tests.jsonnet#L116